### ISSUE TITLE
Avoid the translation of mathematical elements

### ISSUE THREAD

**Issue by ktaaaki** [FilipePS/Traduzir-paginas-web#598]
I want to prevent translation for mathematical elements because it would be misshapen.
This add-on already has the potential to solve that issue. Here is [the **rough** solution](https://github.com/ktaaaki/Traduzir-paginas-web/tree/master/src/contentScript) for it.

### Overview
Translated math elements will appear as follows.
<img width="2637" alt="figure" src="https://user-images.githubusercontent.com/4715386/224910608-02c3a2e8-2a3a-454e-b5e0-a6d755002be7.png">
(the original figure is taken from https://math.stackexchange.com/questions/1454409/show-a-function-is-well-defined-and-continuous?rq=1 .)

By avoiding the translation of math elements, the display will look like the figure on the right.

### The rough solution
Math elements appear mainly in `<math>` tags or in nodes overwritten by MathJax.

The former can be treated as one of [`htmlTagsNoTranslate`](https://github.com/ktaaaki/Traduzir-paginas-web/blob/c227afa0d2f53f98834ecc956cefbff66005a27c/src/contentScript/pageTranslator.js#L296) on `contentScripts/showTranslated.js` or `contentScripts/pageTranslator.js`.

The latter can be [detected by class name](https://github.com/ktaaaki/Traduzir-paginas-web/blob/c227afa0d2f53f98834ecc956cefbff66005a27c/src/contentScript/pageTranslator.js#L380) (e.g. "MathJax" or "math-container"), but I'm not sure if that is the right way.

### Limitations
The inlined math element splits the translated sentence, so it is still not easy to read. Only math contents will be displayed correctly.

**Comment by NWater23** [comment]
Right-click on the MathJax node, then select `Math Settings` > `Math Renderer` > `SVG`.  
Now TWP won't translate text in SVG picture.

> [!NOTE]
You may have to repeat this operation on each sites you view.  
MathJax only save cookies to current domain.

<details>

![图片](https://github.com/FilipePS/Traduzir-paginas-web/assets/128229048/3f143b21-7aab-487c-a852-8e9d8d6037b7)[^1]
![图片](https://github.com/FilipePS/Traduzir-paginas-web/assets/128229048/4cb34f38-010f-499d-9f7a-08a236d41e12)[^2]

</details>

BTW, some sites[^3] are using KaTeX instead of MathJax. Not all of them do things in same way.

[^1]: Screenshot from https://guide.encode.moe/encoding/masking-limiting-etc.html
[^2]: Screenshot from https://math.stackexchange.com/questions/1454409/show-a-function-is-well-defined-and-continuous?rq=1

**Comment by ktaaaki** [comment]
Your method seems to have completely solved the issue. It might have been overlooked before because cookies were not being saved. 
Thank you.
